source/Calamari.GoogleCloudScripting.Tests/GoogleCloudActionHandlerFixture.cs

```diff
@@ -55,13 +55,18 @@ private GoogleStorageObject RetrieveFileFromGoogleCloudStorage(StorageClient cli
 
                 foreach (var result in results)
                 {
-                    if (result.TimeCreated.HasValue)
+                    if (result.TimeCreated.HasValue);
```

Comment:
```suggestion
                    if (result.TimeCreated.HasValue)
```

This semicolon seems unintentional. I think it would terminate the `if` so the next code block executes unconditionally.

Reply:
🤦‍♂️ Cheers for the catch, I've moved the date check inline with this and removed it